tools/myst/host/strace.c

```diff
@@ -88,18 +99,84 @@ int myst_set_strace_filter(
         }
     }
     strace_config->filter = 1;
+
+    if (tokens)
+        free(tokens);
+
     return 0;
 }
 
+/* sets tid filter if 'tid' is true, else sets pid filter */
+int myst_set_strace_id_filter(
```

Comment:
Consider `myst_set_trace_pid_tid_filter`, makes it easier to get what the function does in the callsite.

Reply:
I agree, will make the change